pypogs/hardware_cameras.py

```diff
@@ -248,13 +266,18 @@ def identity(self):
 
         - For model *ptgrey* this is the serial number *as a string*
         - Must set before initialising the device and may not be changed for an initialised device.
+        
+        - For model *ascom*, a driver name may be specified if known, (i.e. DSLR, ASICamera1, ASICamera2, 
+        QHYCCD, QHYCCD_GUIDER, QHYCCD_CAM2, AtikCameras, AtikCameras2, etc), otherwise the ASCOM driver
+        selector will open.
```

Comment:
Does ascom have any way of identifying individual hardware resources? E.g. what happens if I have two cameras connected?